corehq/util/context_processors.py

```diff
@@ -235,20 +235,20 @@ def mobile_experience_hidden_by_toggle(request):
 
 def maintenance_alerts(request):
     active_alerts = MaintenanceAlert.get_active_alerts()
-    if not active_alerts:
```

Comment:
I'd rather have an early return than an extra layer of indentation in the rest of the function, but reasonable developers can disagree on this, esp. since it isn't that long of a function.